data/default.nix

```diff
@@ -45,4 +51,6 @@ in mkDerivation {
   name = "${name}";
   srcs = ./elm-srcs.nix;
   src = ./.;
+  srcdir = "${srcdir}";
+  targets = [${targets}];
```

Comment:
Do people usually write `Main` as exposed in end user apps?

Is there any use compiling each exposed module into js?

Reply:
I don't know really. It depends a bit on the purpose of `elm2nix`. My initial understanding was that it compiled the exposed modules, but apparently by default it builds nothing at all. Does `elm2nix` even make sense for library packages?

Anyway for my personal use, I need to be able to configure some (typically one) target. E.g. like I did here: https://github.com/robx/elm-pointer-draw/. (Or more involved with git dependencies: https://github.com/robx/elm-pointer-draw/tree/capture.)

So, for me, a default empty list of targets would be fine. And the code that parses the list of exposed modules can go. Will make those changes.

Reply:
I think it only makes sense for apps. So something like `$srcdir/Main.hs` sounds best.